utility/env.hpp

```diff
@@ -24,7 +24,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 /**
- *  @file utility/getenv.hpp
+ *  @file utility/env.hpp
```

Comment:
@vaclavblazek can we rename the file? (I.e. can this cause clashes in other projects?)

Reply:
`utility/getenv.hpp` was added, that includes `env.hpp`, but emits `#pragma` deprecation warning